apps/studio/components/interfaces/HomeNew/ServiceStatus.tsx

```diff
@@ -238,15 +200,18 @@ export const ServiceStatus = () => {
                   : 'COMING_UP'
               : isMigrationLoading
                 ? 'COMING_UP'
-                : 'ACTIVE_HEALTHY') as ProjectServiceStatus,
+                : ('ACTIVE_HEALTHY' as ProjectServiceStatus),
```

Comment:
is this cast necessary? based on the previous conditions, kinda looks like it might not be?